charters/wot-wg-charter-draft-2019.html

```diff
@@ -244,28 +582,73 @@ <h3>
+      <p>Produced under <b>Working Group Charter:</b> The  <span class="todo">stable URL of the Working Group charter</span> under which the most recent Reference Draft or Candidate Recommendation was published.
+    </dd>
+              </dl>
+
+            <dt id="wot-smart-home-profile-1-0" class="spec"><a href="#">Web of Things (WoT) Smart Home Profile 1.0</a></dt>
             <dd>
-              <p>This specification defines <i class="todo">[concrete description]</i>.</p>
-
-              <p class="draft-status"><b>Draft state:</b> <i class="todo">[No draft | <a href="#">Use Cases and Requirements</a> | <a href="#">Editor's Draft</a> | <a href="#">Member Submission</a> | <a href="#">Adopted from WG/CG Foo</a> | <a href="#">Working Draft</a>]</i></p>
-              <p class="milestone"><b>Expected completion:</b> <i class="todo">[Q1–4 yyyy]</i></p>
-   <dl><dt>
-    <p class="todo">Per <a href="https://www.w3.org/Consortium/Process/#WGCharter">5.2.6</a>, 
-	    for every Recommendation Track deliverable that continues work on a Working Draft (WD) 
-	    published under any other Charter (including a predecessor group of the same name), 
-	    for which there is an existing Reference Draft or Candidate Recommendation: </p>
+              <p>This specification defines a profile for using the Web of Things in the Smart Home context.
+              </p>
+
+              <p class="draft-status"><b>Draft state:</b> <a href="#">Use Cases</a></p>
+              <p class="milestone"><b>Expected completion:</b> Q2 2021</p>
+              <dl><dt>
       <p><b>Adopted Working Draft:</b> The <span class="todo">title, stable URL, and publication date of the <a href="https://github.com/w3c/charter-drafts/blob/gh-pages/draft-states.md#adopted-draft">Adopted Working Draft</a></span> which will serve as the basis for work on the deliverable.
       <p><b>Reference Draft:</b> The <span class="todo">title, stable URL, and publication date of the most recent <a href='https://github.com/w3c/charter-drafts/blob/gh-pages/draft-states.md#reference-draft'>Reference Draft</a> or Candidate Recommendation</span> which triggered an Exclusion Opportunity per the Patent Process.
         <span class="todo">Exclusion period <b>began</b>; Exclusion period <b>ended</b>.</span> <span class='todo'>(this <a href="https://www.w3.org/2004/01/pp-impl/charter-assistant?wgid=32061">charter assistant</a> helps in producing the list. use the proper <a href='https://www.w3.org/2004/01/pp-impl/'>wgid</a>)</span>
       <p>Produced under <b>Working Group Charter:</b> The  <span class="todo">stable URL of the Working Group charter</span> under which the most recent Reference Draft or Candidate Recommendation was published.
     </dd>
-              </dl></dl>
+              </dl>
 
+            <dt id="wot-discovery-1-0" class="spec"><a href="#">Web of Things (WoT) Discovery 1.0</a></dt>
```

Comment:
Discovery is useful for trusted domains (e.g. behind a firewall), but does not work across the web without a kind of registry or search engine.
We should have very clear use cases and requirements to address these requirements.

Reply:
I'm trying to use "discovery" in the generic sense, and actually am assuming the use of directories as part of it, not broadcast protocols.   In a local environment, a broadcast protocol might be used for the introduction phase.   In a global environment, I might use a search engine.   But both would point at a directory, which would then (after access controls to preserve privacy...) would allow "Exploration".   It is much harder to preserve privacy if you transmit metadata over things like a broadcast protocol so we would want to work with IETF to *discourage* that and use directories generally.   For peer-to-peer, I think we would still want to see the directory network API and access controls being used, even if the "directory" only serves the metadata for a single device.

However, it's clear I was not being clear.   But I was also trying to avoid putting too much "design" into the description.  Let me think about it... I could perhaps give the above as an "example" design. but obviously we need to discuss it more in the WG and in collaboration with IETF especially (resource directories, etc).

Reply:
I modified the description of discovery a bit and explicitly called out support for both local and global discovery.